metsim/metsim.py

```diff
@@ -307,8 +379,11 @@ def setup_output(self, prototype: xr.Dataset=None):
             # Need to convert some parameters to strings
             if k in ['start', 'stop', 'annual', 'mtclim_swe_corr']:
                 v = str(v)
+            elif k in ['state_start', 'state_stop']:
+                # skip
+                continue
             # Don't include complex types
-            if isinstance(v, dict):
+            elif isinstance(v, dict):
```

Comment:
I think I like this being an `if` more, since it's talking about a separate variable than the first `if` statement.  This allows for us to catch more than one situation, and is just a more cautious approach in general.